data/campaigns/Northern_Rebirth/scenarios/05a_02_Dealings.cfg

```diff
@@ -16,10 +16,10 @@
     [story]
         [part]
             music=loyalists.ogg
-            story= _ "Laden with gold and filled with an elation at their victory that was tempered with sadness at the death of their friends, Tallin and his people made their way back to the mines."
+            story= _ "Laden with gold, and filled with both elation at their victory and sadness for their departed comrades, Tallin and his people made their way back to the mines."
```

Comment:
Should probably have a `#po:` comment for "departed comrades". It's clear to a native speaker, but it's also an idiom that some might not have met.

Might also want to comment about the importance of the gold to the later plot, thus why it's appearing before the mention of the dead.